src/Avalonia.Controls/Slider.cs

```diff
@@ -193,7 +193,7 @@ private void MoveToPoint(PointerPoint x)
             var orient = Orientation == Orientation.Horizontal;
 
             var pointDen = orient ? _track.Bounds.Width : _track.Bounds.Height;
-            pointDen += double.Epsilon; // Just add epsilon to avoid divide by zero exceptions.
+            pointDen += MathUtilities.DoubleEpsilon; // Just add epsilon to avoid divide by zero exceptions.
```

Comment:
https://sharplab.io/#v2:EYLgZgpghgLgrgJwgZwLQGUCWBbADgGwgAUEB7AcwSm2QBoYQE4A7AHwGNTnkYACAE1JxghXgA9eAXl4AGAHQyA3AFgAUAAEAjHK0BOABSbeAenEBKFRu17DJ3vokBqAUJEQ5AUVzJM+LmYs1LR1NAyNTB15nACY5aOiZABZEgDYk3WiAVhkAZk08iFQZTRSAxSA

Reply:
https://github.com/AvaloniaUI/Avalonia/blob/954c70257e10ceb42a8e80b5ad1dec49256648ce/src/Avalonia.Controls/Slider.cs#L196-L199

https://github.com/AvaloniaUI/Avalonia/blob/954c70257e10ceb42a8e80b5ad1dec49256648ce/src/Avalonia.Base/Utilities/MathUtilities.cs#L101-L115

Reply:
You are right, I forgot that IEEE spec defines floating point division by zero as producing infinity. As long as our clamp function deals fine with it we won't need that extra epsilon.

Only integer divisions throw exceptions.